### PR TITLE
cherry-pick Add EIP 2930 support to EthTXData (#7696)

### PR DESCRIPTION
**Pull request by stoqnkpL**
Add EIP 2930 support to EthTXData

- Update `EthTXData` with EIP 2930 logic
- Update `EthTXDataTest` with type 1 tests based on existing type 2 tests

**Description**:

**Related issue(s)**:

Fixes #

**Notes for reviewer**:

**Checklist**

- [ ] Documented (Code comments, README, etc.)
- [ ] Tested (unit, integration, etc.)

### CODE REVIEW

**Review by lukelee-sl — APPROVED** [review]
LGTM

File: hedera-node/hedera-mono-service/src/main/resources/bootstrap.properties [hunk 1]
```diff
@@ -112,6 +112,7 @@ hedera.transaction.maxMemoUtf8Bytes=100
 hedera.transaction.maxValidDuration=180
 hedera.transaction.minValidDuration=15
 hedera.transaction.minValidityBufferSecs=10
+hedera.transaction.eip2930.enabled=false
```
Comment (Nana-EC) [inline hedera-node/hedera-mono-service/src/main/resources/bootstrap.properties:115]: We can set this to true now after offline clarification
```suggestion
hedera.transaction.eip2930.enabled=true
```

**Review by Nana-EC — COMMENTED** [review]
Can default to true now

File: hedera-node/hedera-mono-service/src/main/resources/bootstrap.properties (hunk 1 above)
Comment (stoqnkpL) [reply hedera-node/hedera-mono-service/src/main/resources/bootstrap.properties]: updated.

**Review by lukelee-sl — APPROVED** [review]

**Review by netopyr — APPROVED** [review]
LGTM in regards to `hedera-base`

**Pull request merged**